services/media/service/src/generated/messaging/rascal-schema.json

```diff
@@ -407,7 +407,7 @@
         },
         "media-service:cue_point_types:declared": {
```

Comment:
Currently, a change like this would make sure that we receive messages using the updated service ID, but:
1. the previously existed queue` ax-encoding-service:cue_point_types:declare` in media-template-specific vhost will continue to exist.
2. The queue for events, e.g. `media-service:cue_point_types:declared` would have bindings for both service IDs: 
![image](https://github.com/Axinom/mosaic-media-template/assets/117624390/8a669ac5-9d4c-4a01-b664-842a1a1a4261)
   a. This will result in media service receiving a message 2 times because of our backward compatibility logic.

In short - we need to perform some sort of RabbitMQ cleanup to remove obsolete queues and bindings

Reply:
Yes right - the cleanup should be done also as part of this PR - thanks.

Reply:
Was done as part of https://github.com/Axinom/mosaic-media-template/pull/140/files